packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -224,6 +224,42 @@ export default class BuySellStore extends BaseStore {
                         text_size: 'xs',
                     },
                 });
+            } else if (code === api_error_codes.ADVERTISER_SCHEDULE_AVAILABILITY) {
+                general_store.showModal({
+                    key: 'ErrorModal',
+                    props: {
+                        error_message: message,
+                        error_modal_button_text: localize('OK'),
```

Comment:
by default the ErrorModal button text is OK and it does properly translate, so you can remove this
```suggestion
```